merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/services/LocalMissionModelService.java

```diff
@@ -136,7 +136,16 @@ public List<BulkArgumentValidationResponse> validateActivityArgumentsBulk(
       final List<ActivityDirectiveForValidation> activities
   ) throws NoSuchMissionModelException, MissionModelLoadException {
```

Comment:
`NoSuchMissionModelException` isn't getting thrown by this method anymore and can be removed from the signature.

Reply:
Change has been made in the latest commit